lib/screens/order_history.dart

```diff
@@ -95,3 +114,12 @@ class _OrderList extends State<OrderList> {
     Navigator.push(context, orderResponse);
   }
 }
+
+class Order {
```

Comment:
Instead of creating a new class, please add these fields as optional parameters in the already existing "Order" model at models/order.dart.